packages/ui/src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,20 @@
+import "react-loading-skeleton/dist/skeleton.css"
+
+import SkeletonPrimitive from "react-loading-skeleton"
+
+import { useTheme } from "@/theme"
+
+export type SkeletonProps = React.ComponentPropsWithoutRef<
+  typeof SkeletonPrimitive
+>
+
+export const Skeleton: React.FC<SkeletonProps> = (props) => {
+  const { themeProps } = useTheme()
```

Comment:
what's a difference between getToken from utils file and themeProps from the hook?

Reply:
getToken can be used only in SX prop or other styled-components props like our Box component.

Skeleton component is from installed package react-loading-skeleton and the props baseColor and highlightColor dont support that